src/server.ts

```diff
@@ -1,84 +1,462 @@
 import express from "express";
 import cors from "cors";
 import dotenv from "dotenv";
-import {
-  addDummyDbItems,
-  addDbItem,
-  getAllDbItems,
-  getDbItemById,
-  DbItem,
-  updateDbItemById,
-} from "./db";
-import filePath from "./filePath";
-
-// loading in some dummy items into the database
-// (comment out if desired, or change the number)
-addDummyDbItems(20);
+import { Client } from "pg";
+import { INewResource } from "./interfaces";
+import { EmbedBuilder, WebhookClient } from "discord.js";
 
 const app = express();
-
-/** Parses JSON data in a request automatically */
 app.use(express.json());
-/** To allow 'Cross-Origin Resource Sharing': https://en.wikipedia.org/wiki/Cross-origin_resource_sharing */
 app.use(cors());
-
-// read in contents of any environment variables in the .env file
 dotenv.config();
 
-// use the environment variable PORT, or 4000 as a fallback
+const webhookClient = new WebhookClient({
+  id: "1068528197408403516",
+  token: "8DTodD8ydc-IyD5Ycpk9me3fDw_ffuQi_HNxK1VcoKt7iVOaV8wv4U3U3y_dGqGCyVea",
```

Comment:
token should not be available to anyone, only developers. Maybe use an environment variable.